camelot/types/__init__.py

```diff
@@ -301,9 +302,10 @@ class Movie( Entity ):
     
     impl = types.Unicode
 
-    def __init__(self, storage=None, *, max_length=100, **kwargs):
+    def __init__(self, storage, max_length=100, **kwargs):
+        assert isinstance(storage, Storage) or storage is None
```

Comment:
Storage mag hier ook een verplicht argument zijn

Reply:
De storage is hier toch verplicht? of bedoel je storage is None, want dat heb ik gedaan voor de migrations dat je geen storage moet aanmaken voor niets..